bundles/org.openhab.binding.modbus.lambda/src/main/java/org/openhab/binding/modbus/lambda/internal/handler/BoilerHandler.java

```diff
@@ -0,0 +1,585 @@
+                        poller = boilerReg50Poller;
+                        break;
+
+                    default:
+                        poller = null;
+                        break;
+                }
+                if (poller != null) {
+                    logger.trace("Boiler: Es wird gepollt }");
+                    poller.poll();
+                }
+            }
+        } else {
+            // logger.trace("Boiler: handleCommand: Es wird geschrieben, GroupID: {}, command {}",
+            // channelUID.getGroupId(), command);
+            try {
+
+                if (GROUP_BOILER_REG50.equals(channelUID.getGroupId())) {
+                    // logger.trace("Boiler: im BOILER_REG50 channelUID {} ", channelUID.getIdWithoutGroup());
+
+                    switch (channelUID.getIdWithoutGroup()) {
+
+                        case CHANNEL_BOILER_MAXIMUM_BOILER_TEMPERATURE:
+
+                            // logger.trace("Boiler: command: {}", command);
+                            writeInt16(reg50baseadress, getScaledInt16Value(command));
+                            break;
+
+                    }
+                }
+
+            }
+
+            catch (LambdaException error) {
+                if (hasConfigurationError() || getThing().getStatus() == ThingStatus.OFFLINE) {
+                    return;
+                }
+                String cls = error.getClass().getName();
+                String msg = error.getMessage();
+
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        String.format("Error with: %s: %s", cls, msg));
+            }
+        }
+    }
+
+    /**
+     * Initialization: Load the config object of the block Connect to the slave
+     * bridge Start the periodic polling
+     */
+    @Override
+    public void initialize() {
+        config = getConfigAs(BoilerConfiguration.class);
+        // logger.debug("Initializing thing with properties: {}", thing.getProperties());
+
+        startUp();
+    }
+
+    /**
+     * Adresses for the polling registers, used for reading and writing
+     */
+    private int baseadress;
+    private int reg50baseadress;
+
+    /**
+     * This method starts the operation of this handler Connect to the slave bridge
+     * Start the periodic polling1
+     */
+    private void startUp() {
+        if (comms != null) {
+            return;
+        }
+
+        ModbusEndpointThingHandler slaveEndpointThingHandler = getEndpointThingHandler();
+        if (slaveEndpointThingHandler == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, "Bridge is offline");
+            return;
+        }
+
+        try {
+            slaveId = slaveEndpointThingHandler.getSlaveId();
+
+            comms = slaveEndpointThingHandler.getCommunicationInterface();
+        } catch (EndpointNotInitializedException e) {
+            // this will be handled below as endpoint remains null
```

Comment:
mayeb log to debug or trace?

Reply:
C&P from the stiebeleltron-bundle.
I am not capable of changing it.

Reply:
???

Reply:
What do you mean? I opt4ed to add a ```logger.debug("x-y-z- exception message");```